cuda_core/cuda/core/experimental/_linker.py

```diff
@@ -77,13 +77,15 @@ def _lazy_init():
             "fatbin": _nvjitlink.InputType.FATBIN,
             "ltoir": _nvjitlink.InputType.LTOIR,
             "object": _nvjitlink.InputType.OBJECT,
+            "lib": _nvjitlink.InputType.LIBRARY,
```

Comment:
`"library"` would make this fully self-consistent (as is it's `lib` in some places and `library` in others).